ngx-fudis/projects/ngx-fudis/src/lib/components/button/button.component.html

```diff
@@ -31,20 +35,24 @@
 <ng-container *ngIf="!tooltip">
   <button
```

Comment:
Currently tab-key doesn't work at all on Safari. Try adding tabindex=0 here, which makes button accessible with Safari browser

Reply:
Tab key is working fine for me on Safari as well.